community/neatvi/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+sed -i -e '/CFLAGS = /c\' -e 'CFLAGS += -Wall' Makefile
+sed -i s/0600/0666/ ex.c
```

Comment:
Same goes for this one.

Reply:
`neatvi` creates files with mode 0600 by default, I don't like it being so strict. Do you want comments for things like this?

Reply:
Omitting comments is fine so long as there's a written explanation *somewhere* (Comments on this PR for example). This addition may also be temporary if the developer changes it to `0666`.  :+1:

Reply:
Got it.

Also asked the author to make that change.